app.ini

```diff
@@ -1,5 +1,6 @@
 ; app.ini
 [uwsgi]
+strict = true
```

Comment:
Finding uwsgi configuration information is pretty hard. Please include a comment for the blog post you found these at. The PR is usually a good place for this, but the commit message or inline comments are the best.

Reply:
I will add an inline comment, thanks